src/fb.js

```diff
@@ -53,24 +53,34 @@ var {version} = require('../package.json'),
 		return JSON.parse(header['x-page-usage']);
 	},
 	buildRateLimitObjectFromJson = function(rateLimit) {
-		return Object.assign(Object.create(null), {
+		return {
 			callCount: rateLimit['call_count'],
 			totalTime: rateLimit['total_time'],
 			totalCPUTime: rateLimit['total_cputime']
-		});
+		};
 	},
 	stringifyParams = function(params) {
-		var data = {};
+		let data = {};
+		const keys = Object.keys(params);
 
-		for ( let key in params ) {
+		keys.forEach(key => {
 			let value = params[key];
-			if ( value && typeof value !== 'string' ) {
-				value = JSON.stringify(value);
+
+			try {
```

Comment:
What is the rationale for catching the error here? We're not parsing network data, we're serializing developer input. So this seems like the case where we would explicitly want the error to be thrown because the developer did something invalid, instead of silently consuming the error and causing hard to understand bugs.